test/MusicStore.E2ETests/StoreSmokeTests.cs

```diff
@@ -34,7 +34,7 @@ public async Task DefaultLocation_Kestrel()
                     EnvironmentName = "SocialTesting",
                     PublishApplicationBeforeDeployment = true,
                     PreservePublishedApplicationForDebugging = Helpers.PreservePublishedApplicationForDebugging,
-                    TargetFramework = Tfm.NetCoreApp20, // There's only a Store on 2.0
+                    TargetFramework = "netcoreapp3.0",
```

Comment:
@JunTaoLuo is this still relevant?

Reply:
Nope, no stores since 2.2. I don't think these tests are being run on the CI.